src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -304,7 +341,17 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
     @Override
     public Object onTraversalSuccess() {
         if (outputType == CommonCode.OutputType.BQ && shouldWriteLoadStatusStarted) {
-            loadStatus.writeLoadStatusStarted(Long.parseLong(sampleId));
+            loadStatus.writeLoadStatusStarted(sampleId);
+        }
+
+        if (vcfHeaderLineTempCreater != null) {
+            try {
+                vcfHeaderLineTempCreater.apply(allLineHeaders);
+            }catch (IOException ioe) {
+                throw new GATKException("Error writing missing intervals", ioe);
```

Comment:
this feels like an odd error msg, but maybe I'm misunderstanding

Reply:
You're right — fixed.